codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/EntityServiceSourceGenerator.scala

```diff
@@ -160,15 +160,14 @@ object EntityServiceSourceGenerator {
         "com.akkaserverless.javasdk.eventsourcedentity.EventSourcedEntity",
         "com.akkaserverless.javasdk.impl.eventsourcedentity.EventSourcedEntityHandler"))
 
-    val serviceApiOuterClass = service.fqn.parent.javaOuterClassname
-    val outerClassAndState = s"${entity.fqn.parent.javaOuterClassname}.${entity.state.fqn.name}"
```

Comment:
We have been doing too much of string concatenation and repeating it. Good to push that to the internal model! 👍